__docs__/wonder-blocks-button/button.stories.tsx

```diff
@@ -203,6 +135,11 @@ Variants.parameters = {
             story: "There are three kinds of buttons: `primary` (default), `secondary`, and `tertiary`.",
         },
     },
+    chromatic: {
+        // We already have screenshots of other stories that cover more of
+        // the button states
+        disableSnapshot: true,
+    },
```

Comment:
**note:** I'm disabling a bunch of snapshots now that we can centralize visual regression tests in the `All variants` stories.